components/requestvideotask.brs

```diff
@@ -0,0 +1,25 @@
+function init()
+    m.top.functionName = "getcontent"
```

Comment:
The functionName can be 'execute' because that is more descriptive. This is also available for the `requestTask`